hamlpy/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
```

Comment:
This was the problem causing strings to not be well picked by makemessages as this app was not loading well to the `templatize` patch was not being called